### PR TITLE
Monkey mask makes monkey noises like sec hailer

### PR DESCRIPTION
**Pull request by spessbro**
## About The Pull Request

The monkey mask now works like a sec hailer but only makes understandable noise to things that speak monkey. Code bounty by terranaut.

## Why It's Good For The Game

heehoo monkey noises funny

## Changelog
:cl:
add: monkey mask works like a sec hailer
soundadd: added sound for the monkey mask
/:cl:

### CODE REVIEW

File: code/modules/clothing/masks/gasmask.dm [hunk 1]
```diff
@@ -136,6 +136,142 @@
...
+/obj/item/clothing/mask/gas/monkeymask/verb/screech()
+	set category = "Object"
+	set name = "Screech"
+	set src in usr
+	if(!isliving(usr))
+		return
+	if(!can_use(usr))
+		return
+	if(broken_hailer)
+		to_chat(usr, "<span class='warning'>\The [src]'s screeching system is jammed full of bananas paste.</span>")
```
Comment (ATH1909) [inline code/modules/clothing/masks/gasmask.dm]: "bananas paste"

Comment (PKPenguin321) [reply code/modules/clothing/masks/gasmask.dm]: Banana should be singular, if the comment above me is unclear

Comment (spessbro) [reply code/modules/clothing/masks/gasmask.dm]: oops

**Comment by optimumtact** [comment]
go away monkey posters

**Comment by terranaut1** [comment]
https://files.catbox.moe/x4yz4l.jpg very epic

**Comment by kriskog** [comment]
Why would anyone actually want this obnoxious shit in the game. It's literally a mask that lets you scream and annoy people, exactly the reason we aren't able to *scream loudly in the first place.

File: code/modules/clothing/masks/gasmask.dm [hunk 2]
```diff
@@ -136,6 +136,142 @@
 	item_state = "monkeymask"
 	flags_cover = MASKCOVERSEYES
 	resistance_flags = FLAMMABLE
+	actions_types = list(/datum/action/item_action/screech)
+	var/recent_uses = 0
+	var/broken_hailer = 0
```
Comment (actioninja) [inline code/modules/clothing/masks/gasmask.dm]: use boolean defines

File: code/modules/clothing/masks/gasmask.dm [hunk 3]
```diff
@@ -136,6 +136,142 @@
...
+		if(cooldown_special < world.time - 180) //A better cooldown that burns jerks
+			recent_uses = initial(recent_uses)
+
+		switch(recent_uses)
+			if(3)
+				to_chat(usr, "<span class='warning'>\The [src] is starting to smell weird.</span>")
+			if(4)
+				to_chat(usr, "<span class='userdanger'>\The [src] is smelling like burnt peanuts!</span>")
+			if(5) //overload
+				broken_hailer = 1
```
Comment (actioninja) [inline code/modules/clothing/masks/gasmask.dm]: use boolean defines

**Review by actioninja — CHANGES_REQUESTED** [review]
this probably should be a component but it's only in two places so we haven't hit the rule of 3 yet.

File: code/datums/components/crafting/recipes.dm [hunk 4]
```diff
@@ -723,3 +723,11 @@
...
 	category = CAT_MISC
 	tools = list(TOOL_WRENCH, TOOL_WELDER, TOOL_WIRECUTTER)
+
+/datum/crafting_recipe/multiduct
+	name = "Multi-layer duct"
+	result = /obj/machinery/duct/multilayered
+	time = 5
+	reqs = list(/obj/item/stack/ducts = 5)
+	category = CAT_MISC
+	tools = list(TOOL_WELDER)
```
Comment (actioninja) [inline code/datums/components/crafting/recipes.dm]: ???

File: code/modules/clothing/masks/gasmask.dm [hunk 5]
```diff
@@ -136,6 +136,142 @@
 	item_state = "monkeymask"
 	flags_cover = MASKCOVERSEYES
 	resistance_flags = FLAMMABLE
+	actions_types = list(/datum/action/item_action/screech)
+	var/recent_uses = 0
+	var/broken_hailer = FALSE
+	var/gorilla = FALSE
+	var/cooldown_special
```
Comment (actioninja) [inline code/modules/clothing/masks/gasmask.dm]: init to 0 or this will runtime

**Review by actioninja — CHANGES_REQUESTED** [review]

File: code/modules/clothing/masks/gasmask.dm (hunk 5 above)
Comment (spessbro) [reply code/modules/clothing/masks/gasmask.dm]: oki :>

File: code/datums/components/crafting/recipes.dm (hunk 4 above)
Comment (spessbro) [reply code/datums/components/crafting/recipes.dm]: if i leave it out i get a merge conlflict so i'll comply with that

**Review by actioninja — CHANGES_REQUESTED** [review]
That's not how merge conflicts work, literally just checkout the latest example of that file from upstream because you didn't change it

**Comment by PKPenguin321** [comment]
(to do that open a git bash and enter `git checkout upstream/master -- code/datums/components/crafting/recipes.dm`)

**Comment by spessbro** [comment]
haha am stoppido :)

**Comment by spessbro** [comment]
haha i poopied my pants bye bye

**Pull request closed without merging**